entrypoint

```diff
@@ -13,4 +13,4 @@ EOF
   rm /tmp/passwd
 fi
 
-exec jbang $INPUT_JBANGARGS "${@}"
\ No newline at end of file
+exec jbang ${INPUT_JBANGARGS:+"$INPUT_JBANGARGS"} "${@}"
```

Comment:
whats does that syntax mean. ? use input_jbangargs others input_jbangargs ?

Reply:
ah it dont put it there if empty - well it doesn't matter here as they are not quoted here.

They are when listed in action.yml which is what caused this whole thing thus 0.36.1 just listen to env variables to avoid ambiguity.